canonical-web-and-design/vanilla-framework
# Update stylelint to v14

## Done

Updated stylelint to v14 to make sure VSCode extension works with it.
Reordered rules in stylelint config to make it more structured.

## QA

- Make sure CI checks pass
- Pull the branch locally in VSCode
- Make sure stylelint in VSCode works (try reordering some CSS props or break some other rules)
